src/PortingAssistant.Client.Common/Utils/MemoryUtils.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Diagnostics;
+
+namespace PortingAssistant.Client.Common.Utils
+{
+    public static class MemoryUtils
+    {
+        public static void LogMemoryConsumption(ILogger logger)
```

Comment:
If we are logging, can we also log below components too a)Size of the Solutions(being assessed). b)Memory usage on various components (used in the application**) c) Log also various generations objects and large objects (if possible)... ** if possible, not sure

Reply:
Thanks for your comment @birojnayak. 

For `a)` we can for sure include in this PR, we can get the current solution dir and sum the size of all `.cs` files in it and log it, I'll add this in the current PR. 

For `b)` and `c)`, I'm planning to do some memory profiling and analysis in the next couple of days to determine where the most memory consumption is, once we have the result, I will add logs based on that. wdyt?

Reply:
@ShanshanHe please create an ISSUE and tag here in that case for b and c

Reply:
[SIM](https://sim.amazon.com/issues/P49081672)
and corresponding [PR](https://github.com/aws/porting-assistant-dotnet-client/pull/105)